.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature request
+about: Suggest a feature for the project
+labels: feature, incoming
```

Comment:
It is unclear how 'incoming' and 'processed' issues are different. Confirmation that we're interested in the feature in theory? Set assignee and precise milestone? Since we don't have such a triage process now, I would not add any label of this kind.

We can return here after some discussions with @kyukhin and @Mons.

Reply:
label `incoming` was requested by @Mons in a private conversation

Reply:
Agree. We building now such a process, so this label will be useful

Reply:
So we should set it for both feature requests and bug reports?

Reply:
Created the label so.

Reply:
Mons requested label `incoming` for feature requests only.

Reply:
As I heard, the motivation was 'the team anyway catches bugs'. In fact, it is not so. Let's look at the [crashes](https://github.com/tarantool/tarantool/issues?q=is%3Aopen+is%3Aissue+label%3Acrash) for example: 43 open issues. And 221 [bugs](https://github.com/tarantool/tarantool/issues?q=is%3Aopen+is%3Aissue+label%3Abug+) at whole. Our priorities are quite dependent of customers and plans around valuable features. In fact, 'bug' label does not meant that it is always has higher priority than a feature. Sad, but now we're in such a reality.

Despite triage process we'll setup, all new issue should go though it.

So I'll cast @Mons and @kyukhin to decide here as owners of the future process.

If they will agree, let's resolve the discussion with the decision and go ahead. Or just resolve, go ahead and ask colleagues for feedback later. Up to you.